packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -53,6 +53,7 @@
     border: none;
     border-bottom: 1px solid $mdc-textfield-underline-on-light-idle;
     background: none;
+    /* @alternate */
```

Comment:
Would it be better to remove `font-size` from the `@each` a few lines up instead, then we can probably just remove `font-size: inherit` too? (Assuming I'm not being an idiot, and this doesn't cause a display regression)

Reply:
Looked into it a bit, and I'm going to leave as-is for now. The `font-size` in the `@each` above is the correct one, but it is always overriden by this definition here. So the correct fix is to remove this `font-size`, but that might cause regressions and this is just a refactor.

Reply:
Closing this out: I removed the first `font-size` rule since it never applies currently, with a TODO to re-add it later. Thus I can remove the `@alternate` entirely.